libs/@blockprotocol/service/README.md

```diff
@@ -0,0 +1,63 @@
+## Block Protocol – Service Module
+
+This package implements the Block Protocol Service module for blocks and embedding applications.
+
+To get started:
+
+1.  `yarn add @blockprotocol/service` or `npm install @blockprotocol/service`
+1.  Follow the instructions to use the service module as a [block](#blocks) or an [embedding application](#embedding-applications)
+
+## Blocks
+
+To create a `ServiceBlockHandler`, pass the constructor an element in your block, along with any callbacks you wish to register to handle incoming messages.
+
+### React example
+
+For React, we provide a `useServiceBlockService` hook, which accepts a `ref` to an element, and optionally any `callbacks` you wish to provide on initialization.
+
+See `npx create-block-app@latest my-block --template react` for an example.
```

Comment:
I don't think there is such an example and we are unlikely to include one as it requires an API key

Reply:
Thanks for catching, have removed in https://github.com/blockprotocol/blockprotocol/pull/954/commits/10150936fea7d8e7370001884bd8761d04b67280